src/PresentationalComponents/Breadcrumbs/Breadcrumbs.js

```diff
@@ -60,7 +60,7 @@ const Breadcrumbs = ({ current }) => {
         <Breadcrumb ouiaId="detail">
           {items.map((oneLink, key) => (
             <BreadcrumbItem key={key}>
-              <Link to={oneLink.navigate}>{oneLink.title}</Link>
```

Comment:
The constructing of the `navigate` in `buildBreadcrumbs` seems to be borked.

Reply:
I had to bring the Link back, otherwise it would break when I try to go back to the dashboard using breadcrumbs